cowprotocol/cowswap
# Pixel tracking

# Summary

This PR adds Meta and Linkedin pixel tracking for following events
- when the page (url) is changed
- when the app is loaded
- when an account is connected
- when trade is posted

For more context [this is the document](https://docs.google.com/spreadsheets/d/1GFg-gXmBboLGLeLsTGA_7FxmuCB3P0xNbdwHhEEe7d4/edit?pli=1#gid=1015340718) where the events are defined

# To test
As I can see the only way to test this would be to check the requests in dev-tools Network tab

Meta
- when the app is loaded, filter events by word `initiateCheckout` and there should be request
- when the account is connect, filter by `Contact`
- when the order is placed, filter by `Lead`

Linkedin
- when the app is loaded, filter events by `10759506`
- when the account is connect, filter by `10759514`
- when the order is placed, filter by `10759522`
